deploy/etos-logarea/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21-alpine AS build
+FROM golang:1.22-alpine AS build
```

Comment:
The version will need to be in sync with requirements in go.mod.

Since iut, sse and logarea share the same go.mod file, they will likely require the same version of Go.